kratos/python/add_processes_to_python.cpp

```diff
@@ -305,6 +307,20 @@ void  AddProcessesToPython(pybind11::module& m)
             .def_readonly_static("CALCULATE_EXACT_DISTANCES_TO_PLANE", &VariationalDistanceCalculationProcess<3,SparseSpaceType,LocalSpaceType,LinearSolverType>::CALCULATE_EXACT_DISTANCES_TO_PLANE)
     ;
 
+    py::class_<ParallelDistanceCalculatorProcess<2>, ParallelDistanceCalculatorProcess<2>::Pointer, Process>(m,"ParallelDistanceCalculatorProcess2D")
+        .def(py::init<ModelPart&, Parameters>())
+        .def(py::init<Model&, Parameters>())
+        .def(py::init<ModelPart&, const Variable<double>&, const Variable<double>&, const unsigned int, const double>())
+        .def(py::init<ModelPart&, const Variable<double>&, const Variable<double>&, const unsigned int, const double, const bool>())
```

Comment:
```suggestion
```

better only use the `Parameters` constructor, same as the 3D version

Reply:
I understand, it just made my life easier to adapt the new interface in all the places its used. The idea was to make it into some steps, but I can try to solve it now.

Reply:
Ah I see
well if i makes the migration easier then how abt adding this CTor but with a deprecation warning?
e.g.
https://github.com/KratosMultiphysics/Kratos/blob/f180cdd09e6426b228df56f7c3424f0b1d1d9ee2/kratos/python/add_strategies_to_python.cpp#L635-L638

Reply:
yep, sure!

Reply:
I meant to add a runtime warning, not a compile-time one (or maybe both if the Core compile clean)

Reply:
i see, well I think it will be easier to just remove the constructor, it will not be much different

Reply:
done